docs/content/docs/getting-started/build.md

```diff
@@ -167,17 +167,27 @@ cp -rf prompt-templates/ggml-gpt4all-j.tmpl models/
 curl http://localhost:8080/v1/models
 
 curl http://localhost:8080/v1/chat/completions -H "Content-Type: application/json" -d '{
-     "model": "ggml-gpt4all-j",
+     "model": "phi-2.Q2_K",
      "messages": [{"role": "user", "content": "How are you?"}],
      "temperature": 0.9 
    }'
 ```
 
 #### Troublshooting mac
 
-If you encounter errors regarding a missing utility metal, install `Xcode` from the App Store.
-If completions are slow, ensure that `gpu-layers` in your model yaml matches the number of layers from the model in use (or simply use a high number such as 256).
-If you a get a compile error: `error: only virtual member functions can be marked 'final'`, reinstall all the necessary brew packages, clean the build, and try again.
+1. If you encounter errors regarding a missing utility metal, install `Xcode` from the App Store. 
+2. After the installation of Xcode, if you receive a xcrun error `'xcrun: error: unable to find utility "metal", not a developer tool or in PATH'`. You might have installed the Xcode command line tools before installing Xcode, the former one is pointing to an incomplete SDK. 
```

Comment:
added one additional entry, as I encountered it myself during `make build`

here is the detailed thread https://github.com/gfx-rs/gfx/issues/2309